Update to deps.edn and clearing some test

davidpham87 opened the pull request:
Hello,

I desired to use your library for one of my project, and I could not use it with `deps.edn`, so I quickly changed it. And since your were nice enough to write test, I also cleared some of them so that all test passes (I basically hacked the date time regex).

Thanks a lot for your work.

davidpham87 committed:
feat: deps update and deps.edn conversion

- Using `deps.edn` for managing dependencies
- Solving bug for time range and dates parsing.
- Debug of test.

davidpham87 committed:
docs

davidpham87 committed:
docs: fix deps.edn coordinate

bnbeckwith commented:
@davidpham87, thank you for the PR.  I'll need to update my knowledge on how =deps.edn= works to fully approve the request.  I'll add some comments to your PR soon and we can go from there.

bnbeckwith commented on README.org:
```diff
@@ -3,23 +3,33 @@
 ** Installation
 
-   If you are using =lein=, then just add it as a dependency.
+   Using =deps.edn=
```
Can we have both =lein= and =deps.edn=?

davidpham87 replied:
Probably yes, I can add it back. I could not find where your jar was located in maven/clojars though.

bnbeckwith commented on README.org:
```diff
@@ -3,23 +3,33 @@
 
 #+BEGIN_SRC clojure
-  [bnbeckwith.com/orgmode "0.7.2"]
+  {:deps {orgmode {:git/url "https://github.com/davidpham87/orgmode"
```
Should this be a link to my repository?

davidpham87 replied:
Yes it should be your repository, once we get the merge.

bnbeckwith commented on src/orgmode/inline.clj:
```diff
@@ -40,25 +40,28 @@
 
 ;; ### Timestamp Elements
 
+;; hack to make it work with ranges
```
Pedantic: Is it a _hack_ or a _fix_?

davidpham87 replied:
I don't really know, honestly :-). It is hack in the sense that I don't why you get an intermediate grouping (my regexfu is limited), this is why it is hack. Empirically, it provides the requested results though.

bnbeckwith commented on src/orgmode/inline.clj:
```diff
@@ -147,3 +151,12 @@
       (make-elem (fmt-re "~")   (fmt-create :code))))
+
+
+(comment
```
I think that we can remove this comment and turn it into a test?

davidpham87 replied:
Okay.

bnbeckwith reviewed (changes requested):
Thanks for the PR!

I think that your documentation commits can be squashed together. Otherwise, I left a few comments inline so that I can understand all of the parts.

davidpham87 committed:
docs and chore: Tests and documentation

- Renaming of testing namespace to make them discoverable by cider/emidje.
- Addition of inline testing.

davidpham87 commented:
Voilà! I hope the change answers your requests.

davidpham87 committed:
bug: enclosing-plain-list

- The `not` had wrong argument and I guess it was meant to be `not=`

bnbeckwith reviewed (approved):
Looks good.  Thanks!
